internal/librarian/state.go

```diff
@@ -150,13 +151,28 @@ func findServiceConfigIn(path string) (string, error) {
 	return "", nil
 }
 
+func saveLibrarianConfig(repoDir string, librarianConfig *config.LibrarianConfig) error {
```

Comment:
Drive-by comment: does this preserve formatting and comments? I suspect not.

Part of the point of separating state from config was that config would be hand-maintained, and *could* contain comments (which then wouldn't be wiped). I'm concerned that we may lose that as part of this change.